tests/e2e/dpa_deployment_suite_test.go

```diff
@@ -381,7 +381,7 @@ var _ = Describe("Configuration testing for DPA Custom Resource", func() {
 	}
 
 	awsTests := []TableEntry{
-		Entry("AWS Without Region No S3ForcePathStyle with BackupImages false should succeed", InstallCase{
+		Entry("AWS Without Region No S3ForcePathStyle with BackupImages false should succeed", Label("aws"), InstallCase{
```

Comment:
I would rather see this label added in a for loop instead since that's the whole point of a separate `awsTests` declaration. But I am aware this is not possible right now.
https://github.com/onsi/ginkgo/pull/974 might allow this in the future.